src/DayColumn.js

```diff
@@ -140,6 +153,45 @@ class DaySlot extends React.Component {
     );
   }
 
+  renderAvailabilities = () => {
+    let {
+      availabilities,
+      availabilityComponent,
+      availabilityWrapperComponent: AvailabilityWrapper,
+      availabilityStartAccessor,
+      availabilityEndAccessor,
+      min,
+      date,
+    } = this.props;
+    let AvailabilityComponent = availabilityComponent;
+    let styledAvailabilities = getStyledAvailabilities({
+      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin, date
+    });
+
+    return styledAvailabilities.map(({availability, style}, index) => {
+      let { height, top } = style;
+
+      return AvailabilityWrapper && (
+        <AvailabilityWrapper key={index}>
+          <div
+            className='rbc-availability'
+            style={{
+              top: `${top}%`,
+              height: `${height}%`,
+            }}
+            title='Provider Availability'
```

Comment:
I would remove the word "Provider". See my other comment about how this is a general calendar component, not a part of the EMR codebase. If there's no explicit desire to show specific text on hover, I might even suggest omitting this entirely for now. Otherwise, we'd only want to have a really generic title here and give consumers the option to override the title via something similar to the `titleAccessor` prop (e.g. `availabilityTitleAccessor`).

Reply:
Actually, yeah, good call. There's no need to show title information here. If a title is needed, it can be handled by the availability view component.